src/V3Coverage.cpp

```diff
@@ -39,6 +42,18 @@ class CoverageVisitor final : public VNVisitor {
     // TYPES
     using LinenoSet = std::set<int>;
 
+    struct CoverTerm final {
+        AstNodeExpr* m_exprp;  // Expression branch term
+        bool m_objective;  // Term objective
+        std::string m_emitV;  // V3EmitV string for cover point comment
+        CoverTerm(AstNodeExpr* exprp, bool objective, const string& emitV)
+            : m_exprp{exprp}
+            , m_objective{objective}
+            , m_emitV(emitV) {}
+    };
+    using CoverExpr = std::list<CoverTerm>;
+    using CoverExprs = std::list<CoverExpr>;
```

Comment:
Is there an advantage of a list over a std::deque?  std::list does a lot of allocs.

Reply:
I don't see an obvious problem with as it is, just wanted to make sure it made sense, so we can leave it.